src/kvstore/RocksEngine.cpp

```diff
@@ -461,5 +433,69 @@ ResultCode RocksEngine::createCheckpoint(const std::string& name) {
     return ResultCode::SUCCEEDED;
 }
 
-}  // namespace kvstore
-}  // namespace nebula
+ErrorOr<ResultCode, std::string> RocksEngine::backupTable(
+    const std::string& name,
+    const std::string& tablePrefix,
+    std::function<bool(const folly::StringPiece& key)> filter) {
+    auto backupPath = folly::stringPrintf(
```

Comment:
How to tell the difference between multiple backup? BTW, why we don't write one sst for on space instead of multiple

Reply:
Just because the order of the keys must be guaranteed when exporting sst, and the space id in some of our tables is not a prefix, it is easier to export sst by table.

Reply:
Yes, it may need to refactor the MetaServiceUtils code about table here.

Reply:
Or we need to redesign the table structure so that one type of table has nothing to do with spaceid and one type of table can get all the data directly from the spaceid prefix.